src/services/mentees.js

```diff
@@ -779,4 +780,207 @@ module.exports = class MenteesHelper {
 			return error
 		}
 	}
+
+	/**
+	 * Get entities and organization filter
+	 * @method
+	 * @name getFilterList
+	 * @param {String} tokenInformation - token information
+	 * @param {Boolean} queryParams - queryParams
+	 * @returns {JSON} - Filter list.
+	 */
+	static async getFilterList(queryParams, tokenInformation) {
+		try {
+			let result = {
+				organizations: [],
+				entity_types: {},
+			}
+
+			let organization_ids = []
+			const organizations = await this.getOrganizationIdBasedOnPolicy(tokenInformation.id)
+			if (organizations.success && organizations.result.length > 0) {
+				organization_ids = [...organizations.result]
+
+				const defaultOrgId = await getDefaultOrgId()
+				if (defaultOrgId && !organization_ids.includes(defaultOrgId)) {
+					organization_ids.push(defaultOrgId)
+				}
+
+				let findOrganizationAndEntities = false
```

Comment:
const findOrganizationAndEntities = organization_ids.length > 0 ? true: false;

Should be cleaner and equivalent